mailbox/pom.xml

```diff
@@ -82,35 +82,4 @@
         <url>http://issues.apache.org/jira/browse/MAILBOX</url>
     </issueManagement>
 
-    <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    </properties>
-
-    <repositories>
-        <!-- apache.snapshot is needed for james-project SNAPSHOT, see JAMES-1470 -->
-        <repository>
-            <releases>
-                <enabled>false</enabled>
-            </releases>
-            <id>apache.snapshots</id>
```

Comment:
we don't need these for example if you rely temporary on a non released version of mime4j?

Reply:
> we don't need these for example if you rely temporary on a non released version of mime4j?

Why wouldn't we defined snapshot repositories in `james-project/pom.xml` ?

Having it at this level in the project hierarchy makes no sense...